### PR TITLE
Overview

### PR DESCRIPTION
**Pull request by THEO-184**
pulling request from my overview branch.

### CODE REVIEW

**Comment by beganovich** [comment]
Please remove the draft status of the PR when it is ready for review.

**Comment by THEO-184** [comment]
> Please remove the draft status of the PR when it is ready for review.

Please I don't get what you  mean by this. I don't know why it is been tagged as Draft.

**Comment by beganovich** [comment]
I made it a draft because it's not ready for review.

File: src/pages/products/View.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  if (isLoading) {
+    return (
+      <Default>
+        <Container>
+          <div className="flex justify-center">
+            <Spinner />
+          </div>
+        </Container>
+      </Default>
+    );
```
Comment (beganovich) [inline src/pages/products/View.tsx:84]: This is not convenient with the rest of the application.

https://github.com/invoiceninja/ui/blob/main/src/pages/clients/show/components/Contacts.tsx

File: src/pages/products/View.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  }
+
+  return (
+    <Default>
```
Comment (beganovich) [inline src/pages/products/View.tsx:88]: Breadcrumbs can be passed into the component.
https://github.com/invoiceninja/ui/blob/main/src/pages/clients/show/Client.tsx#L83

File: src/pages/products/View.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+import { Container } from '../../components/Container';
+import { Default } from '../../components/layouts/Default';
+import { Spinner } from '../../components/Spinner';
+import { Breadcrumbs } from 'components/Breadcrumbs';
+import { Tab, Tabs } from '../../components/Tabs';
+
+export function View() {
+  const [t] = useTranslation();
+  const { id } = useParams();
+  const [ProductDetails, setProductDetails] = useState<ProductDetails>(
```
Comment (beganovich) [inline src/pages/products/View.tsx:26]: This variable is out of the convention of the project. Please use camelCase for variables.

File: src/pages/products/View.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    <Default>
+      <Container>
+        <Breadcrumbs pages={pages} />
+        <Tabs tabs={tabs} className="mt-6" />
+
+        <Card>
+          <Element leftSide={t('price')}>
+            ${ProductDetails?.price?.toFixed(2)}
+          </Element>
+          <Element leftSide={t('Description')}>Product Description</Element>
```
Comment (beganovich) [inline src/pages/products/View.tsx:97]: This has no purpose. The product description should be pulled from the resource itself, not hardcoded.

**Review by beganovich — CHANGES_REQUESTED** [review]

**Comment by beganovich** [comment]
Looks good, slight changes are needed.

There's a problem with how you handle tabs, we don't want duplicated page for this, rather re-rendering outlet, instead of a whole new page. An example of this is available on the client show page.

**Pull request merged**